File: VB/02_ParagraphAndText/ModifyStyleOfFirstFoundText.vb
Imports Spire.Presentation
Imports Spire.Presentation.Drawing

Namespace ModifyStyleOfFirstFoundText
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub btnRun_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnRun.Click
			'Create an instance of presentation document
			Dim ppt As New Presentation()
			ppt.LoadFromFile("..\..\..\..\..\..\Data\TextTemplate.pptx")

			'Find first "Spire"
			Dim text As String = "Spire"
			Dim textRange As TextRange = ppt.Slides(0).FindFirstTextAsRange(text)

			'Modify the style
			textRange.Fill.FillType = FillFormatType.Solid
			textRange.Fill.SolidColor.Color = Color.Red
			textRange.FontHeight = 28
			textRange.LatinFont = New TextFont("Calibri")
			textRange.IsBold = TriState.True
			textRange.IsItalic = TriState.True
			textRange.TextUnderlineType = TextUnderlineType.Double
			textRange.TextStrikethroughType = TextStrikethroughType.Single

			'Save the document
			Dim result As String = "Result.pptx"
			ppt.SaveToFile(result, FileFormat.Pptx2013)
			PresentationDocViewer(result)
		End Sub

	Private Shared Sub PresentationDocViewer(ByVal fileName As String)
			Try
				Process.Start(fileName)
			Catch
			End Try
	End Sub

	End Class
End Namespace

File: CS/02_ParagraphAndText/ModifyStyleOfFirstFoundText.cs
using System;
using System.Collections.Generic;
using System.Drawing;
using System.Windows.Forms;
using Spire.Presentation;
using Spire.Presentation.Drawing;

namespace ModifyStyleOfFirstFoundText
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();            
        }

        private void btnRun_Click(object sender, EventArgs e)
        {
            //Create an instance of presentation document
            Presentation ppt = new Presentation();
            ppt.LoadFromFile(@"..\..\..\..\..\..\Data\TextTemplate.pptx");

            //Find first "Spire"
            string text = "Spire";
            TextRange textRange = ppt.Slides[0].FindFirstTextAsRange(text);
            
            //Modify the style
            textRange.Fill.FillType = FillFormatType.Solid;
            textRange.Fill.SolidColor.Color = Color.Red;
            textRange.FontHeight = 28;
            textRange.LatinFont = new TextFont("Calibri");
            textRange.IsBold = TriState.True;
            textRange.IsItalic = TriState.True;
            textRange.TextUnderlineType = TextUnderlineType.Double;
            textRange.TextStrikethroughType = TextStrikethroughType.Single;

            //Save the document
            string result = "Result.pptx";
            ppt.SaveToFile(result, FileFormat.Pptx2013);
            PresentationDocViewer(result);
}
	
	private static void PresentationDocViewer(string fileName)
        {
            try
            {
                System.Diagnostics.Process.Start(fileName);
            }
            catch { }
        }

    }
}